template/faas-flow/.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
Probably better not commit the `.idea` folder and add an entry to the `.gitignore` file

Reply:
Fixed